### ISSUE TITLE
ConnectionError on OSX with version 0.9.2

### ISSUE THREAD

**Issue by mheadd** [datacats/datacats#81]
I'm seeing an error similar to [this one](https://github.com/boxkite/datacats/issues/60). Running:

OSX 10.10.3
Python 2.7.9
boot2docker 1.6.1
datacats 0.9.2 

```
~$ datacats create myckan
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/bin/datacats", line 9, in <module>
    load_entry_point('datacats==0.9.2', 'console_scripts', 'datacats')()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/pkg_resources.py", line 339, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/pkg_resources.py", line 2470, in load_entry_point
    return ep.load()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/pkg_resources.py", line 2184, in load
    ['__name__'])
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/datacats/cli/main.py", line 40, in <module>
    from datacats.cli import create, manage, install, pull, purge, shell, deploy
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/datacats/cli/create.py", line 11, in <module>
    from datacats.environment import Environment, DatacatsError
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/datacats/environment.py", line 22, in <module>
    from datacats.docker import (web_command, run_container, remove_container,
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/datacats/docker.py", line 36, in <module>
    _version_client.version()['ApiVersion'])
  File "/Library/Python/2.7/site-packages/docker/client.py", line 1108, in version
    return self._result(self._get(url), json=True)
  File "/Library/Python/2.7/site-packages/docker/client.py", line 106, in _get
    return self.get(url, **self._set_request_timeout(kwargs))
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/requests/sessions.py", line 477, in get
    return self.request('GET', url, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/requests/sessions.py", line 465, in request
    resp = self.send(prep, **send_kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/requests/sessions.py", line 573, in send
    r = adapter.send(request, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/requests/adapters.py", line 415, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', error(2, 'No such file or directory'))
```

Not really sure what I'm doing wrong. Any help appreciated.

**Comment by JackMc** [comment]
Did you run:
```
boot2docker init
boot2docker up
```
? This is what usually happens to me when I forget that, or a long hang.

**Comment by mheadd** [comment]
```boot2docker up``` produces:

<code>
Your environment variables are already set correctly.
</code>

**Comment by deniszgonjanin** [comment]
Just to make sure your docker works, can you do:
```
docker run busybox ls
```

**Comment by mheadd** [comment]
Yup, works as expected.

```
Status: Downloaded newer image for busybox:latest
bin
dev
etc
home
lib
lib64
linuxrc
media
mnt
opt
proc
root
run
sbin
sys
tmp
usr
var
```

**Comment by wardi** [comment]
Would you paste the output of
```bash
python -c'from docker.utils import kwargs_from_env; print kwargs_from_env()'
```

**Comment by mheadd** [comment]
```
{'tls': <docker.tls.TLSConfig object at 0x105ce4310>, 'base_url': 'https://192.168.59.103:2376'}
```

**Comment by wardi** [comment]
Looks reasonable. Do you have an HTTPS proxy configured by any chance?

**Comment by wardi** [comment]
One more command to try, please :-)
```bash
python -c'from docker.utils import kwargs_from_env; from docker import Client; print Client(**kwargs_from_env()).version()'
```

**Comment by mheadd** [comment]
```
{u'KernelVersion': u'4.0.2-boot2docker', u'Arch': u'amd64', u'ApiVersion': u'1.18', u'Version': u'1.6.1', u'GitCommit': u'97cd073', u'Os': u'linux', u'GoVersion': u'go1.4.2'}
```

**Comment by wardi** [comment]
That's great! so it's a problem with the version we're requesting.

Very strange that we can't reproduce that on our end, but try changing /Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/datacats/docker.py line 34 from:
```python
_version_client = Client(version=MINIMUM_API_VERSION, **_docker_kwargs)
```
to:
```python
_version_client = Client(**_docker_kwargs)
```

**Comment by mheadd** [comment]
That seems to have done it. Thanks!

**Comment by wardi** [comment]
@mheadd thanks. I'd like to put out a release with this fix. Would you verify that this fix works for you [latest-0.9 branch](https://github.com/boxkite/datacats/tree/latest-0.9)

**Comment by mheadd** [comment]
@wardi Verified. Works without issue.

**Comment by wardi** [comment]
released 0.9.3 with this fix
